test/unit/setup/node.js

```diff
@@ -14,10 +14,10 @@ if (!global.document || !global.window) {
   var jsdom = require('jsdom').jsdom;
```

Comment:
@paulfalgout the old setup file used was this. I was thinking of keeping this and getting rid of the `test/setup/node.js` instead so not to break the existing setup. Would one be better than the other?